backend/src/modules/auth/auth.controller.ts

```diff
@@ -53,15 +53,15 @@ export class AuthController {
   }
 
   // add @MinimumRole(UserRole.SOMETHING) to secure the route with a minimum role
-  @Get('min-role-secured')
+  @Get('min-role-secured-user')
```

Comment:
Don't we want enums for this?

Reply:
These endpoints are for testing and developing - it shows how decorators should be used, nothing more.